x-pack/plugins/dataset_quality/server/routes/data_streams/routes.ts

```diff
@@ -92,7 +100,43 @@ const degradedDocsRoute = createDatasetQualityServerRoute({
   },
 });
 
+const dataStreamDetailsRoute = createDatasetQualityServerRoute({
+  endpoint: 'GET /internal/dataset_quality/data_streams/details',
```

Comment:
could we just change this route to something like `'GET /internal/dataset_quality/{datasetName}/details'`? 
At the end I think we will be using this route to just query the information of an specific dataset, wdyt?